src/client.ts

```diff
@@ -373,6 +373,25 @@ export class GenesysCloudWebrtcSdk extends (EventEmitter as { new(): StrictEvent
     }
   }
 
+  /**
+   * Start a video conference using a meeting id. Not supported for guests.
+   *  Conferences can only be joined by authenticated users
+   *  from the same organization.
+   *
+   *  `initialize()` must be called first.
+   *
+   * @param meetingId meetingId of the conference to join.
+   *
+   * @returns a promise with an object with the newly created 'conversationId'
+   */
+  async startVideoMeeting (meetingId: string): Promise<{ conversationId: string }> {
```

Comment:
I separated this from a normal video conference partly just for the sake of development. I kinda like having it separate like this, but feel free to combine it with `startVideoConference` if that's preferred.

Reply:
If its possible to easily determine the difference between a `meetingId` and `roomJid` then it may be simpler to combine them so people consuming this don't get confused as to what they each do. 

I'm open to leaving it as is though, but if we do can we update the readme with it?